### ISSUE TITLE
Mnist Experiment

### ISSUE THREAD

**Issue by lin88lin8850** [dbbert/dfn#6]
I really appreciate your work. That's amazing. 

I'm really interested in your model and try to realize it in tensorflow. However, the loss doesn't decrease. I feel very frustrated. I couldn't figure it out. Could you mind release a tensorflow version of mnist experiment?

Thank you very much.

Best Regards,
Jacqueline

**Comment by dbbert** [comment]
Hi,

I have added a basic tensorflow implementation of the bouncing mnist video prediction experiment in [experiment_bouncingMnistOriginal_tensorflow.ipynb](https://github.com/dbbert/dfn/blob/master/experiment_bouncingMnistOriginal_tensorflow.ipynb). I only implemented training, no evaluation on the test set. There might be some small differences compared to the Lasagne implementation, so we cannot guarantee that you reach the same accuracy that is reported in the paper.

Best,
Bert
